docs/tutorials/text-classification-tutorial.mdx

```diff
@@ -0,0 +1,295 @@
+
+---
+
+- Now, it's time to create your prediction model!
+
+```sql
+CREATE MODEL mindsdb.hf_test
+PREDICT PRED
+USING
+engine = 'huggingface',
+task = 'text-classification',
+model_name = "Hate-speech-CNERG/bert-base-uncased-hatexplain",
+input_column = 'tweet',
+labels = ['hate_speech', 'neither', 'offensive'];
+```
+
+| hf_test | huggingface | mindsdb | true | 1 | generating | [NULL] | PRED | up_to_date | 23.3.4.0 | [NULL] | [NULL] | {'target': 'PRED', 'using': {'task': 'text-classification', 'model_name': 'Hate-speech-CNERG/bert-base-uncased-hatexplain', 'input_column': 'tweet', 'labels': ['hate_speech', 'neither', 'offensive']}}  | [NULL] |
+
+
+## Checking The Status of a Predictor
+
+- Next up, check the status of your predictor:
+
+```sql
+SELECT *
+FROM mindsdb.models 
+WHERE name = 'hf_test';
+```
+
+| hf_test | huggingface | mindsdb | 1 | complete | [NULL] | PRED | up_to_date | 23.3.5.0 | [NULL] | [NULL] | {'target': 'PRED', 'using': {'task': 'text-classification', 'model_name': 'Hate-speech-CNERG/bert-base-uncased-hatexplain', 'input_column': 'tweet', 'labels': ['hate_speech', 'neither', 'offensive']}} | [NULL] | [NULL] | [NULL] | [NULL] | 2023-03-29 17:15:38.683663 |
+
+---
+
+`DESCRIBE PREDICTOR mindsdb.hf_test;`
+
+```sql
+{"input_column":"tweet","labels":["hate_speech","neither","offensive"],"labels_map":{"hate speech":"hate_speech","normal":"neither","offensive":"offensive"},"max_length":512,"model_name":"Hate-speech-CNERG/bert-base-uncased-hatexplain","target":"PRED","task":"text-classification","task_proper":"text-classification"}
+```
+
+## Training a Predictor
+
+Now that your model is finished generating, you can go ahead and test it out! ***Note: In order to confirm your new model's efficiency, you will most likely use some language you don't like or agree with here. Just keep in mind, you can ultimately use this highly effective technology to do good! Do not follow the tutorial if you intent to use this malicously.***
+
+***Write the actual offensive word when you run these queries, for example 'bitch', not b****. The words below that include stars were inputted as offensive words that are commonly associated with hate speech on the internet. They have been censored with astracts for this tutorial. 
+
+Test Query Examples:
+
+```sql
+SELECT *
+FROM mindsdb.hf_test
+WHERE text = 'I like you. I love you.';
+```
+
+| PRED | PRED_explain | text |
+| ---- | ------------ | ---- |
+| neither | {"hate_speech":0.03551715984940529,"neither":0.7747423648834229,"offensive":0.18974047899246216} | I like you. I love you. |
+
+---
+
+```sql
+SELECT *
+FROM mindsdb.hf_test
+WHERE text = 'I hate n******'
+```
+
+| PRED | PRED_explain | text |
+| ---- | ------------ | ---- |
+| hate_speech | {"hate_speech":0.82469242811203,"neither":0.05819767713546753,"offensive":0.11710995435714722} | I hate n******' |
+
+---
+
+```sql
+SELECT *
+FROM mindsdb.hf_test
+WHERE text = 'You are a dumb w****';
```

Comment:
Please remove all offensive/vulgar words and examples from this tutorial. MindsDB documentation shouldn't contain any negative language or hate speech.

Reply:
@alissatroiano 
Actually, as discussed with @ZoranPandovski, we want to avoid having such examples in the docs. You could keep examples such as 'I like you' or 'I don't like you'. It may not show the full potency of the model, but we want to stick to the 'polite hate speech'.